code/game/machinery/doors/windowdoor.dm

```diff
@@ -252,6 +252,15 @@
 	health = 300.0 //Stronger doors for prison (regular window door health is 200)
 
 
+/obj/machinery/door/window/briefing
+	name = "reinforced safety windoor"
+	icon = 'icons/obj/doors/windoor.dmi'
+	icon_state = "leftsecure"
+	base_state = "leftsecure"
+	req_access = list(ACCESS_MARINE_BRIDGE)
+	health = 3000
```

Comment:
this seems excessive, stick it to 1k or so if possible. Also give this a different description to hint the sturdiness.